core/src/main/java/io/grpc/ChannelImpl.java

```diff
@@ -317,6 +351,16 @@ public void transportTerminated() {
         }
       }
     }
+
+    private boolean clearActiveTransport() {
```

Comment:
Do you still want this as a separate function, even though it is now only used one place?